crates/astria-bridge-withdrawer/src/withdrawer/mod.rs

```diff
@@ -0,0 +1,330 @@
+use std::{
+    net::SocketAddr,
+    sync::Arc,
+    time::Duration,
+};
+
+use astria_core::primitive::v1::asset;
+use astria_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+use tokio::{
+    select,
+    sync::oneshot,
+    task::{
+        JoinError,
+        JoinHandle,
+    },
+    time::timeout,
+};
+use tokio_util::sync::CancellationToken;
+use tracing::{
+    error,
+    info,
+};
+
+pub(crate) use self::state::StateSnapshot;
+use self::{
+    ethereum::Watcher,
+    state::State,
+    submitter::Submitter,
+};
+use crate::{
+    api,
+    config::Config,
+};
+
+mod batch;
+mod ethereum;
+mod state;
+mod submitter;
+
+pub struct Service {
```

Comment:
Call this `WithdrawerBridge` following all our other services.

Reply:
that cause a clippy name repetitions error since this module is already called `withdrawer` and this isn't a bridge, so i'd prefer not to name it that